src/components/ifxToggle/ifxToggle.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/IfxToggle',
+  args: {
+    checked: true,
+  },
+  argTypes: {
+
+    checked: {
+       control: 'boolean',
```

Comment:
Why is the `checked boolean` defined twice?

Reply:
It isn't - one is the default, one is the definition